src/modules/settings/index.js

```diff
@@ -154,7 +154,7 @@ class SettingsModule {
         cdn.get('privacy.html').then(data => $('#bttvPrivacy').html(data));
 
         api.get('changelog')
-            .then(({changelog}) => changelog.map(({version, publishedAt, body}) => changelogEntryTemplate(version, publishedAt, body)))
+            .then(changelog => changelog.map(({version, publishedAt, body}) => changelogEntryTemplate(version, publishedAt, body)))
             .then(releases => $('#bttvChangelog .bttv-changelog-releases').html(releases.join('')));
```

Comment:
breaking change is we cannot `.html` this as we are not serving html which is pre-escaped any longer. need to render the plain text into what we want (should hotlink the issue #s back to github and parse new lines)